syft/cataloging/filecataloging/tags.go

```diff
@@ -0,0 +1,6 @@
+package filecataloging
+
+const (
+	// FileTag should be used to identify catalogers that are file-based.
```

Comment:
non-blocking: `file-based` is sort of confusing. Does this comment mean to say, "... used to identify catalogers that provide information about files rather than packages"?